src/utils.tsx

```diff
@@ -1,3 +1,8 @@
+import { Cell, ICellModel, isCodeCellModel, CodeCell } from "@jupyterlab/cells";
```

Comment:
clean up these multiimports. I suggest installing a typescript linting extension to do this for you.

Reply:
I already have the typescript linting ext installed in vscode. I have removed unneeded imports, but while I was experimenting, I was using them and forgot to remove.

Reply:
I have a linter, I just wasn't sure if I'd be needing the imports later and forgot to remove them.